### PR TITLE
Create browser directories and add a Firefox Addon

### PR DESCRIPTION
**Pull request by miketaylr**
I wanted to be cool and have a `shared/` directory and symlink things, but apparently that offends the Firefox Addon SDK, so it's basically just a copypasta job for the content script. So heads up if you ever have to change it to change it everywhere.

### CODE REVIEW

File: firefox/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "picture-in-play",
+  "title": "Picture in Play",
+  "id": "jid1-BmBEdNsy9Y1EnQ",
+  "icon": "data/icon48.png",
+  "icon64": "data/icon64.png",
+  "description": "Indicates whether the `picture` element is in use on a page.",
+  "author": "Mat",
+  "license": "MIT",
```
Comment (miketaylr) [inline firefox/package.json:9]: I have no idea how you want to license this, so I picked MIT. :cactus:

**Comment by miketaylr** [comment]
OK, apparently symlinks don't work in Chorme either, https://code.google.com/p/chromium/issues/detail?id=27185. Shrugs.

**Comment by miketaylr** [comment]
Hmm, question: Do you want to include the .xpi binary file in the firefox directory... or possibly have a `bin/` directory to keep .xpi, .crx, etc.? Or not include it at all and people can just install the SDK and package it themselves? Any of those wfm.

**Comment by Wilto** [comment]
I’d keep the browser-specific bundle dealies in the directory for each, I think. Lotta people (me) aren’t gonna know which is which at a glance.

**Comment by miketaylr** [comment]
Alright cool. So if you merge this the firefox addon is in the directory. And then you can commit the Chrome one to chrome/. (And then I can make an Opera one to appease Thor).

**Comment by Wilto** [comment]
thx bb

**Pull request merged**